pkg/apis/resolution/v1alpha1/resolution_request_types.go

```diff
@@ -78,9 +79,44 @@ type ResolutionRequestStatusFields struct {
 	// of the requested resource in-lined into the ResolutionRequest
 	// object.
 	Data string `json:"data"`
+	// Source is the source reference of the remote data that records the url, digest
+	// and the entrypoint.
+	Source *ConfigSource `json:"source"`
```

Comment:
@abayer do we need to add this to the alpha remote resolution types or just the beta ones?

Reply:
Looks like we do since the resolvers use v1alpha1